frontend/src/initialToolData.tsx

```diff
@@ -146,7 +146,7 @@ const _initialToolData: Tool[] = [
     features: ['manual virtual environment creation'],
     name: 'virtualenv',
     toolDescription:
-      'A tool for creating isolated virtual Python environments. This is essentially a 3rd party package that duplicates the standard libary venv module. It predates venv, and works for all Python versions, including < 3.5 when venv was introduced.',
+      'A tool for creating isolated virtual Python environments. This is a predecessor of the Python 3.5+ standard library venv module, with improvements such as vastly faster virtualenv creation added in subsequent versions.',
```

Comment:
I would like to be more clear that venv should be preferred unless using an old version of python or if speed is a concern.

Reply:
And any other caveats such as the one @jhermann mentioned

Reply:
Why do you think venv "should be preferred"?

Reply:
Because it is part of the standard library and it works fine.

Reply:
Because more general awesomeness.

Reply:
Actually, I think `virtualenv` is much more preferable due to speed and active maintenance.